### DISCUSSION TITLE
Enquiry

### DISCUSSION THREAD

**Discussion by Trailblazer007** [Open-Science-Community-Saudi-Arabia/Open-Science-Community-in-Saudi#49] [Outreachy 2022] [answered]
Hello everyone, please does anyone have a link to a recording of the last twitter space that was held on **Monday, 24th October**? 
I'd appreciate it!

**Comment by jesutobi** [comment, marked as answer]
i do ,lemme help you with
https://twitter.com/outreachy/status/1584559899035385856?t=ARPR_-Xmb-kdaJqX-oPutQ&s=08

**Comment by BatoolMM** [comment]
Thank you @jesutobi!

**Reply by jesutobi** [reply]
@BatoolMM you are welcome
